modules/Db2/scripts/install_Db2.sh

```diff
@@ -12,113 +12,281 @@
 ###############################################################################
 
 CUR_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-OC_CMD=oc
+
+C_DB2UCLUSTER_DB2U="c-db2ucluster-db2u"
+C_DB2UCLUSTER_RESTORE_MORPH="c-db2ucluster-restore-morph"
+SUBCRIPTION_NAME="db2u-operator"
 
 echo
 echo "Creating project ${DB2_PROJECT_NAME}..."
-${OC_CMD} new-project "${DB2_PROJECT_NAME}"
-${OC_CMD} project "${DB2_PROJECT_NAME}"
+kubectl create namespace "${DB2_PROJECT_NAME}"
+oc project "${DB2_PROJECT_NAME}"
```

Comment:
This needs to be changed to `kubectl ns`

Reply:
Done